scripts/send-trigger-requests.sh

```diff
@@ -12,7 +12,7 @@ do
   # Generate a trigger request
   http_code=$(curl -fsSL -o /dev/null -w "%{http_code}" -X POST \
     -H 'Content-Type: application/json' \
-    --data '{"triggerName":"e0326da3de4b3d4ef4d193907afb82bf9afb938daccea445cbca747bb79c9139:NoOp:noOp","party":"alice","applicationId": "test-app-id"}' \
+    --data '{"triggerName":"e0326da3de4b3d4ef4d193907afb82bf9afb938daccea445cbca747bb79c9139:NoOp:noOp","party":"observability","applicationId": "test-app-id"}' \
```

Comment:
Nit: you might want to name this party in a more specific way, e.g., `observabilityExample` or similar.

Reply:
Renamed party to `observabilityExample`

Reply:
You're right — sorry, forgot about that.